ultimatejobweb/models.py

```diff
@@ -13,6 +13,7 @@ class Job(models.Model):
     job_title = models.CharField(max_length=128)
     description_url = models.URLField()
 
-    def delete_data():
+    @classmethod
+    def delete_data(cls):
         Job.objects.all().delete()
```

Comment:
@BarBenNaim see the previous review comments:
you should call the method like that,
cls.objects.all().delete()